### ISSUE TITLE
Node LTS not included in engine "node" versions

### ISSUE THREAD

**Issue by radumardale** [eslint/eslint#10851]
**Tell us about your environment**

* **ESLint Version:** 5.5.0
* **Node Version:** 8.9.3
* **npm Version:** none - used `yarn@1.9.4`

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
yarn add eslint
```

**What did you expect to happen?**
I was expecting `eslint` to install 

**What actually happened? Please include the actual, raw output from ESLint.**

```bash
error eslint@5.5.0: The engine "node" is incompatible with this module. Expected version "^6.14.0 || ^8.10.0 || >=9.10.0".
error Found incompatible module
```
I understand what the problem is but I'm using [node LTS](https://nodejs.org/en/blog/release/v8.9.0/) - 8.9.3.

Shouldn't node LTS be supported?

**Comment by not-an-aardvark** [comment]
Hi,

Node 8 is in LTS, and ESLint supports any release after 8.10.0 on the Node 8 release line.

Node 8.9.3 is an older release on the Node 8 LTS line that we don't support. The way Node LTS releases work is that users on LTS versions typically expected to update to the latest release for their LTS line in order to get security updates. For example, the latest release on the Node 8 LTS line is Node 8.11.4, which fixes some [security vulnerabilities](https://nodejs.org/en/blog/release/v8.11.4/) found in earlier versions of Node. I would recommend upgrading to Node 8.11.4 in order to use ESLint.

In other words, "Node 8" is an LTS line, but "Node 8.9" is not; in order to continue getting long-term support from the Node project, users of Node 8.9.3 would be expected to upgrade to something like Node 8.11.4.

**Comment by radumardale** [comment]
Thank you very much for the details. 
I'll first upgrade `node`

**Comment by presmini** [comment]
Hi @not-an-aardvark . I am trying to build off the docker image node:carbon which is currently at 8.12.0. I am recieving the error 

error eslint@5.5.0: The engine "node" is incompatible with this module. Expected version "^6.14.0 || ^8.10.0 || >=9.10.0".
error Found incompatible module

Is eslint@5.5.0 supposed to be compatible with node 8.12.0?

**Comment by not-an-aardvark** [comment]
Yes. Are you sure your Node version is actually 8.12.0?

Note that the message isn't actually coming from ESLint (I assume it's coming from your package manager). If it's reporting an error when the Node version is satisfied (v8.12.0 is in the range `"^6.14.0 || ^8.10.0 || >=9.10.0"`), then it's possible that this would be a bug in the package manager.

**Comment by presmini** [comment]
Thanks for the quick response @not-an-aardvark . I've confirmed that the docker container installs node 8.12.0. I'm using yarn 1.9.4 for the package management. So you are thinking that yarn may not be honoring the range correctly? I haven't seen that before.

**Comment by presmini** [comment]
@not-an-aardvark you are correct that the problem was my node version. the current version of docker node:carbon image includes node 8.12, but I had a 9 month old cached version which included 8.9. That fixed it when I cleared the cache!

Thanks for the help!
